language-support/hs/bindings/README.md

```diff
@@ -18,3 +18,24 @@ grpc-haskell-core. The default `cabal.project` file contains an
 example of how to do this.
 
 The main package for the ledger bindings is called `daml-ledger`.
+
+There is currently no separate documentation for the haskell ledger bindings,
+over and above the existing ledger API doc.
+
+The [.proto files](/ledger-api/grpc-definitions/com/digitalasset/ledger/api/v1)
+are the best primary source of truth for the API. The [Haskell
+bindings](/language-support/hs/bindings/src/DA/Ledger/Services) match closely
+the names of the services and RPCs, but of course you get much better
+[types](/language-support/hs/bindings/src/DA/Ledger/Types.hs).
+
+The entry point is at [DA.Ledger](/language-support/hs/bindings/src/DA/Ledger.hs).
+
+To use the bindings in this repo, you need the following `BUILD` dep:
+```
+"//language-support/hs/bindings:hs-ledger",
```

Comment:
I would make the `in this repo` part bold and maybe point out explicitly that this is only for this repo since I don’t expect that most people will use it in the repo.

Reply:
Good point. I've also switched the order, which I  believe reads better as a logical sequence: how the code is structured, how to use in repo, how to use out of repo. (As opposed to previous commit which had how to use outside, how code is structured, how to use inside.)